### PR TITLE
tag via tasks

### PR DESCRIPTION
**Pull request by grosser**

### CODE REVIEW

**Comment by grosser** [comment]
@gregorym as alternative to https://github.com/gregorym/bump/pull/43

**Comment by apanzerj** [comment]
i haz green build :-D

File: lib/bump/tasks.rb [hunk 1]
```diff
@@ -13,10 +13,18 @@
...
-
     task bump do
       run_bump.call(bump, {})
     end
+
+    if bump != "current"
+      namespace bump do
+        desc "bump and tag #{bump}"
+        task :tag do
+          run_bump.call(bump, :tag => true)
```
Comment (grosser) [inline lib/bump/tasks.rb]: stupid hound ... this is still tested on ruby 1.8 :D

**Comment by gregorym** [comment]
Duplicate of #43

File: lib/bump/tasks.rb (hunk 1 above)
Comment (gregorym) [inline lib/bump/tasks.rb]: I agree!!
I removed it.

**Pull request closed without merging**